src/routes/v2/project_creation.rs

```diff
@@ -748,11 +749,15 @@ async fn project_create_inner(
             }
         }
 
+        let organization_id = project_create_data.organization_id.map(|id| id.into());
+        insert_project_create_event(project_id, organization_id, &current_user, transaction)
+            .await?;
```

Comment:
the project creation event should not happen here. Users should only be sent notifications if a project has been approved / has a public status. You can put this in the code where we send discord webhooks to the public channel (in project edit)

Reply:
See disc